.github/workflows/static.yml

```diff
@@ -34,3 +34,15 @@ jobs:
         uses: OskarStark/php-cs-fixer-ga@2.16.3.1
         with:
           args: --dry-run --diff-format udiff
+
+  psalm:
+    name: Psalm
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Psalm
+        uses: psalm/psalm-github-actions@master
```

Comment:
This is unfortunate that we can't pin. I'd actually rather we just loaded PHP then downloaded a phar, so we can pin the tool version, if upstream don't want to do tags.